pkg/server/cloud/gce/instances.go

```diff
@@ -528,12 +528,14 @@ func (c *gceClient) GetImage(spec cloud.BootImageSpec) (cloud.Image, error) {
 			creationTime = &ts
 		}
 	}
+	diskSize := int32(resp.DiskSizeGb)
 	// TODO: these values seem to be reversed?
 	return cloud.Image{
 		ID:           resp.Name,
 		Name:         resp.SelfLink,
 		RootDevice:   "",
 		CreationTime: creationTime,
+		VolumeDiskSize: &diskSize,
```

Comment:
We should make sure to test this on an actual GKE cluster.

Reply:
I'm happy to do this, I'll probably need a bit of helping in setting up the cluster

Reply:
> I'm happy to do this, I'll probably need a bit of helping in setting up the cluster

Sure, let's talk about it once this PR wraps up.

Reply:
as we discussed on slack, I tested on GCE and it looks good, resolving :+1: